src/definitions/modules/calendar.js

```diff
@@ -1483,7 +1483,8 @@ $.fn.calendar.settings = {
 
   error: {
     popup: 'UI Popup, a required component is not included in this page',
-    method: 'The method you called is not defined.'
+    method: 'The method you called is not defined.',
+    dateValue: 'Invalid date value for calendar'
```

Comment:
I think this should be `invalid: 'Invalid date value'` so it can potentially be used for more then one error.